docs/content/users/quickstart.md

```diff
@@ -341,7 +341,76 @@ Here's a quickstart instructions for a number of different environments:
     
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
+    
+    ### Using Vite
+    The default Laravel Vite template will not work with DDEV. To make DDEV and Vite play nicely a few additional steps are required.
+
+    #### Setup a Vite server
+    Install the ddev-viteserve addon.
```

Comment:
Add a courtesy link, use “add-on” rather than “addon”—docs use both but the hyphenated variant is more frequent. _(Makes note to self.)_

```suggestion
    Install the [ddev-viteserve](https://github.com/torenware/ddev-viteserve) add-on.
```